src/services/pin/pin-model.js

```diff
@@ -40,7 +45,10 @@ const pinSchema = new Schema({
   level: String,
   location: {
     type: { type: String, enum: 'Point', default: 'Point' },
-    coordinates: { type: [Number], default: DEFAULT_LAT_LONG },
+    coordinates: {
+      type: [Number],
+      default: [defaultLocation.long, defaultLocation.lat],
```

Comment:
We should bounce back to default location constant.
Otherwise, if a user forgets to set this in config, the whole app will fail.